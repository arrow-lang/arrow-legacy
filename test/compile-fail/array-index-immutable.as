let arr = [1, 2];
arr[1] = 50; // error
[1, 5][0] = 30; // error
