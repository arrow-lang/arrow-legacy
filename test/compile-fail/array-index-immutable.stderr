[0;37mtest/compile-fail/array-index-immutable.as:2.1-6: [0;31merror: [1;37millegal left-hand side expression[0m
[0;37mtest/compile-fail/array-index-immutable.as:3.1-9: [0;31merror: [1;37millegal left-hand side expression[0m
